lang/cs/Org.Apache.REEF.Evaluator/Evaluator.cs

```diff
@@ -141,7 +142,7 @@ private static IConfiguration ReadEvaluatorConfiguration(string evaluatorConfigF
 
             AvroConfigurationSerializer serializer = new AvroConfigurationSerializer();
             var classHierarchy = TangFactory.GetTang()
-                .GetClassHierarchy(new string[] { typeof(ApplicationIdentifier).Assembly.GetName().Name });
+                .GetClassHierarchy(new string[] { typeof(ApplicationIdentifier).GetTypeInfo().Assembly.GetName().Name });
```

Comment:
`Type.Assembly` is not supported in `.NET Core + Platform Extensions, Version=1.0`. The recommended solution is to access via `TypeInfo.Assembly`.